packages/core/src/logger/apiLogger.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  HttpRequest,
+  HttpResponse,
+  ApiLoggerInterface,
+  LoggerInterface,
+  Level,
+} from '../coreInterfaces';
+import { LoggingOptions } from './loggerOptions';
+import { NullLogger } from './nullLogger';
+
+export class ApiLogger implements ApiLoggerInterface {
+  private readonly _logger: LoggerInterface;
+
+  constructor(loggingOp: LoggingOptions) {
```

Comment:
`ApiLogger` constructor argument should be of the type `LoggingConfiguration` instead of `LoggingOptions`.